.github/workflows/procedure-lint.yml

```
@@ -20,9 +20,12 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup Node
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.12.0
         uses: actions/setup-node@v4
```

YAMLlint (1.38.0)

[error] duplication of key "uses" in mapping

(key-duplicates)